src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -9792,4 +9925,33 @@ export class StudyViewPageStore
             this.studyIds[0] === 'heme_onc_nsclc_genie_bpc'
         );
     }
+
+    @action
```

Comment:
seems odd that we would be adding these general methods as part of this PR.  lets discuss

Reply:
is this used?